<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1331</width>
    <height>776</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1331</width>
    <height>776</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1331</width>
    <height>776</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>1230</width>
      <height>463</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="srcImg">
       <property name="minimumSize">
        <size>
         <width>571</width>
         <height>461</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>571</width>
         <height>461</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>128</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QComboBox" name="Algorithm">
         <item>
          <property name="text">
           <string>SSR</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>MSR</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>MSRCR</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>128</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="dstImg">
       <property name="minimumSize">
        <size>
         <width>571</width>
         <height>461</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>571</width>
         <height>461</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>570</y>
      <width>1161</width>
      <height>117</height>
     </rect>
    </property>
    <property name="title">
     <string>参数调整</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>1071</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="maxScale">
        <item>
         <widget class="QLabel" name="scaleLabel">
          <property name="text">
           <string>最大尺度：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="maxScaleSilde">
          <property name="maximum">
           <number>500</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSpinBox" name="maxScaleBox">
        <property name="maximum">
         <number>500</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>18</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="scaleNum">
        <item>
         <widget class="QLabel" name="scaleNumLabel">
          <property name="text">
           <string>尺度数：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="scaleNumSlide">
          <property name="maximum">
           <number>5</number>
          </property>
          <property name="pageStep">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSpinBox" name="scaleNumBox">
        <property name="maximum">
         <number>5</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="ChooseImage">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>490</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>选择图片</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ProcessingImage">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>490</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>处理图片</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>530</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="dstContrast">
     <property name="text">
      <string>方差：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="dstContrastValue"/>
   </widget>
   <widget class="QSplitter" name="splitter_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>530</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="srcEntropy">
     <property name="text">
      <string>信息熵</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="srcEntropyValue"/>
   </widget>
   <widget class="QSplitter" name="splitter_4">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>530</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="dstEntropy">
     <property name="text">
      <string>信息熵</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="dstrEntropyValue"/>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>530</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="srcContrast">
     <property name="text">
      <string>方差：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="srcContrastValue"/>
   </widget>
   <widget class="QSplitter" name="splitter_5">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>530</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="srcArticulation">
     <property name="text">
      <string>清晰度：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="srcArticulationValue"/>
   </widget>
   <widget class="QSplitter" name="splitter_6">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>530</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="srcArticulation_2">
     <property name="text">
      <string>清晰度：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="dstArticulationValue"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1331</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
